bugguide/bugguide
# #82 Create bgusers features to export roles and permissions

### Description

Create /sites/all/modules/bgusers
in order to export roles and permissions

qq: How do I enable bgusers? is there a file for enabling modules? should  enable bgusers in 
`drush en -y bgusers` https://github.com/bugguide/bugguide_beta_utils/blob/main/sync_beta.sh#L70 ?

### Testing steps

1. Log in as administrator
2. Enable bgusers
3. Roles should have permissions in /admin/people/permissions


Correct permissions for each specific role will be done in ticket https://github.com/bugguide/bugguide/issues/83
